<?xml version="1.0"?>
<launch>
  <arg name="sim" default="false" />
  <arg name="enable_position_pid" default="true" />
  <arg name="enable_velocity_pid" default="true" />
  <arg name="cascaded_pid" default="false" />

  <node pkg="controls" exec="controls" name="controls" output="screen" launch-prefix="gdb -ex run --args">
    <param name="sim" value="$(var sim)" />
    <param name="enable_position_pid" value="$(var enable_position_pid)" />
    <param name="enable_velocity_pid" value="$(var enable_velocity_pid)" />
    <param name="cascaded_pid" value="$(var cascaded_pid)" />
  </node>
</launch>